ui/ui-components/components/record/SampleRecordCard.tsx

```diff
@@ -266,20 +268,20 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
         </LinkButton>
       );
     }
-    if (!hideViewAllRecords && hasRecords) {
```

Comment:
It should show the button in config mode.
Let's still show the button either way because this page can be stale. Just have it say "View all Records" - even "View All 0 Records" is fine.

Reply:
So I guess just `if (!hideViewAllRecords) {`